keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/AdminController.kt

```diff
@@ -41,4 +42,11 @@ class AdminController(
   )
   fun getManagedApplications() =
     adminService.getManagedApplications()
+
+  @PostMapping(
+    path = ["/recheck/{resourceId}"]
+  )
+  fun triggerRecheck(resourceId: String) {
+    adminService.triggerRecheck(resourceId)
```

Comment:
Not a review breaker - but maybe it worth adding a test for the recheck?

Reply:
i have a test for the underlying function that it calls, which is where the logic is